modules/twinkleblock.js

```diff
@@ -1040,6 +1040,14 @@ Twinkle.block.blockPresetsInfo = {
 		reasonParam: true,
 		reason: 'Email [[WP:Harassment|harassment]]',
 		summary: 'You have been [[WP:PB|blocked]] from emailing other editors for [[WP:Harassment|harassment]]'
+	},
+	'uw-aepblock': {
+		autoblock: true,
+		nocreate: false,
+		pageParam: false,
+		reason: '[[WP:Arbitration enforcement|Arbitration enforcement]]',
+		reasonParam: true,
```

Comment:
should add a decisionParam to support DS topic codes?

Reply:
Yeah, on my todo list, especially now that there will be two.  Couple ways to do it, will ponder.